tests/filecheck/transforms/convert_onnx_to_linalg.mlir

```diff
@@ -31,10 +43,10 @@
 %res_gemm= "onnx.Gemm"(%t5, %t6, %t7) {onnx_node_name = "/Gemm", "alpha" = 1.000000e+00 : f32, "beta" = 1.000000e+00 : f32, "transA" = 0 : si64, "transB" = 1 : si64}: (tensor<1x320xf32>, tensor<50x320xf32>, tensor<50xf32>) -> tensor<1x50xf32>
 
 // CHECK-NEXT:  %t5, %t6, %t7 = "test.op"() : () -> (tensor<1x320xf32>, tensor<50x320xf32>, tensor<50xf32>)
-// CHECK-NEXT:  %3 = tensor.empty() : tensor<320x50xf32>
-// CHECK-NEXT:  %4 = linalg.transpose ins(%t6:tensor<50x320xf32>) outs(%3:tensor<320x50xf32>) permutation = [1, 0]
+// CHECK-NEXT:  %6 = tensor.empty() : tensor<320x50xf32>
```

Comment:
In order to avoid these spurious changes, I'd recommend using regex patterns:
```suggestion
// CHECK-NEXT:  %{{.*}} = tensor.empty() : tensor<320x50xf32>
```
You can also `pip install filecheckize` and use it with `--mlir-anonymize` to convert MLIR IR to filechek tests, something like this:
`xdsl-opt file.mlir -p my-awesome-pass | filecheckize --mlir-anonymize`

Reply:
oops i merged it, ill keep these in mind for future, thank you

Reply:
oops i merged it, ill keep these in mind for future, thank